src/differentials/thunks.jl

```diff
@@ -45,18 +45,25 @@ julia> t()()
 ### When to `@thunk`?
 When writing `rrule`s (and to a lesser exent `frule`s), it is important to `@thunk`
 appropriately.
-Propagation rule's that return multiple derivatives are not able to do all the computing themselves.
- By `@thunk`ing the work required for each, they then compute only what is needed.
+Propagation rule's that return multiple derivatives may not hae all deriviatives used.
```

Comment:
Typos: 
rule's -> rules
hae -> have

(Not sure why GitHub won't make a direct suggestion, sorry)

Reply:
```suggestion
Propagation rules that return multiple derivatives may not have all deriviatives used.
```